pyblish_lite/model.py

```diff
@@ -268,11 +302,12 @@ def setData(self, index, value, role):
         else:
             self.dataChanged.emit(index, index, [role])
 
-    def update_with_result(self, result):
+    def update_with_result(self, result, action=False):
```

Comment:
This is the only thing making me hesitate.

Some day, me or someone else will try and use this function and wonder "Why does `update_with_result()` take a result *and an action*?".

But I think it's safe to look past, so long as we have this conversation on record and that it will at some point be fixed. :)